ArpHeader/AmKernel/locks.h

```diff
@@ -0,0 +1,172 @@
+/*
```

Comment:
You need to use the private header from the Haiku headers directory instead of adding it to this repository.

Reply:
The reason I borrowed the locks.h from Haiku repository is that as I borrowed also MultiLocker.cpp and MultiLocker.h (the MultiLocker class in Haiku is derived from the same Be code that is in  Sequitur repo but updated a lot.) It was easier to borrow the updated class from haiku than to fix the old one to work on 64 bits. I included the locks.h in here as if it changes in Haiku and is not compatible anymore, that wouldn't break the MultiLocker class. But I can remove the header from here and use the system one.

Reply:
You have to use the system one because it is using kernel and libroot functions, and so if it breaks MultiLocker, it will need to be adjusted.